lib/circular_bottom_navigation.dart

```diff
@@ -204,7 +206,7 @@ class _CircularBottomNavigationState extends State<CircularBottomNavigation>
                 child: Text(
                   widget.tabItems[pos].title,
                   textAlign: TextAlign.center,
-                  style: TextStyle(fontWeight: FontWeight.bold, color: widget.tabItems[pos].color),
+                  style: widget.labelStyle.copyWith(color: widget.tabItems[pos].color),
```

Comment:
As you see here we get the color from each TabItem,
then you can replace the color with a TextStyle inside each TabItem, and set it directly here,
with this scenario, firstly it is more robust, and we just need to fill a TextStyle and pass to each TabItem,  and we can determine different TextStyles for each TabItem,
then please remove the TextStyle from circular_bottom_navigation.dart, and it in tab_item.dart replace the existing color with a TextStyle, and use it here as textStyle.

Reply:
I think I'm going to fork this off then, I completely disagree, a tab item should not be dictating the styling of the overall widget. Aside from colour, I can't think of a single use case where label styling would be completely different between tabs.

Reply:
For future reference, closing pull requests in unnecessary and makes moving forward very difficult.

I'll keep my fork separate.

Reply:
For future reference, closing pull requests is unnecessary and makes moving forward very difficult. If you've got feed back you can simply keep the pull request open and provide a comment, any additional commits will update the pull request automatically.

Reply:
Just to confirm, you're suggesting that the indicator colour come from the TextStyle?

Reply:
what kind of difficulty do you mean?
you can reOpen it again whenever you want,
I'm suggesting to change `tab_item.dart` class and rename `color` to `circleColor`,
then alongside the `title` add a `TextStyle titleStyle;` (to hold data about representing the title),
then use this style in the `circular_bottom_navigation.dart` instead of hard coded `TextStyle`, search it and find it, then replace it with `widget.tabItems[pos].titleStyle`,
then you have to update the sample in `main.dart` to add the titleStyle on each TabItems, and make them bold by default.
is it clear?